components/style/values/computed/font.rs

```diff
@@ -286,6 +287,17 @@ pub type FontVariantLigatures = specified::VariantLigatures;
 /// Use VariantNumeric as computed type of FontVariantNumeric
 pub type FontVariantNumeric = specified::VariantNumeric;
 
+/// Use FontSettings as computed type of FontFeatureSettings
+pub type FontFeatureSettings = FontSettings<FontSettingTagInt>;
+
+impl FontFeatureSettings {
+    #[inline]
+    /// Default value of `font-feature-settings` as `normal`
+    pub fn normal() -> FontFeatureSettings {
```

Comment:
Given this is a `FontSettings` variant, we could try to move this to `FontSettings`, but that can be a followup PR.

Reply:
Great! It should be moved under `FontSettings`; after this PR merged, I'll send one to fix it!